### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by JyothiNandyala**
Adding the javascript1 week2 homework.

### CODE REVIEW

File: git/week1/eventapplication.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+var weekday = ["Sunday" ,"Monday" ,"Tuesday" ,"Wednesday" ,"Thursday" ,"Friday" ,"Saturday"]
+var date = new Date();
+var day = weekday[date.getDay()];
+console.log(day);
+var eventHeldDay=4;
+function getEventWeekday(number){
+  
+  let eventDay=((date.getDay()+number)+7)%7;
```
Comment (aina21) [inline git/week1/eventapplication.js:9]: use const, and I don't understand why do you plus with 7?

File: git/week1/candyhelper.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+
+let boughtCandyPrices =[]
```
Comment (aina21) [inline git/week1/candyhelper.js:2]: const

File: git/week1/candyhelper.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+function addCandy(candyType ,weight){
+    if(candyType==="Sweet"){
+       let  price=weight*0.5;
```
Comment (aina21) [inline git/week1/candyhelper.js:6]: you use a price out of scope, so maybe it is better to defined before if

File: git/week1/candyhelper.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+
+let boughtCandyPrices =[]
+
+function addCandy(candyType ,weight){
+    if(candyType==="Sweet"){
```
Comment (aina21) [inline git/week1/candyhelper.js:5]: maybe switch is more readable here

File: git/week1/candyhelper.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+     
+    }
+  
+}
+addCandy("sweet", 20);
+addCandy("Chocolate", 20);
+addCandy("Toffee", 20);
+addCandy("Chewing-gum", 20);
+
+let amountToSpend =Math.random() * 100;
```
Comment (aina21) [inline git/week1/candyhelper.js:28]: const

File: git/week1/candyhelper.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+console.log(amountToSpend);
+let totalPrice=0;
+for(let i=0;i<boughtCandyPrices.length;i++){
+    totalPrice=totalPrice+boughtCandyPrices[i];
+    console.log(totalPrice);
+}
+
+function canBuyMoreCandy(){
+if(totalPrice>amountToSpend){
```
Comment (aina21) [inline git/week1/candyhelper.js:37]: What about time the totalprice is equal to the amountToSpend.

**Review by aina21 — CHANGES_REQUESTED** [review]
Good job 👍

**Comment by JyothiNandyala** [comment]
Hi Ida,

i have changed my comments and pushed to github,could you please  check and
let me know.

Thanks&Regards
Jyothi.

**Comment by aina21** [comment]
you have unrelated files for week1 in this PR, check your file path. you may get conflict if you merge it.

**Pull request merged**